docs/sitegen.yaml

```diff
@@ -208,6 +208,14 @@ backend:
               items:
                 - includes:
                     - "se/aot/*.adoc"
+            - title: "Fault Tolerance"
+              pathprefix: "/se/faulttolerance"
+              glyph:
+                type: "icon"
+                value: "warning"
+              items:
+                - includes:
+                    - "se/faulttolerance/*.adoc"
         - title: "Helidon MP"
```

Comment:
Is this meant to be MP here?

Reply:
Fault Tolerance SE should be the last entry for SE. The next line is where MP starts as I understand it.